frontend/core/widgets/NodeStylePicker/EmojiTab/index.module.css

```
@@ -0,0 +1,64 @@
+.emojiPicker :global(.epr-emoji-list) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)